platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -47,28 +52,46 @@
 
   private static final Path BIN_DIR = Paths.get("bin");
 
+  /*
+   Files included in an import operation when upgrading to a newer version.
+  */
+  private static final List<String> SYSTEM_PROPERTIES_TO_PRESERVE =
+      ImmutableList.of("solr.password", "migration.supported.versions");
+
+  private static final Path CUSTOM_SYSTEM_PROPERTIES_PATH =
+      Paths.get("etc", "custom.system.properties");
+
+  private static final List<Path> UPGRADEABLE_SYSTEM_PATHS =
+      ImmutableList.of( //
+          Paths.get("etc", "users.properties"), Paths.get("etc", "users.attributes"));
+
+  /*
+   Files (in addition to the above) that are included under normal export and import operations
+   within the same product version.
+  */
   private static final List<Path> REQUIRED_SYSTEM_PATHS =
       ImmutableList.of( //
           Paths.get("etc", "ws-security"),
           Paths.get("etc", "system.properties"),
-          Paths.get("etc", "custom.system.properties"),
+          CUSTOM_SYSTEM_PROPERTIES_PATH,
           Paths.get("etc", "startup.properties"),
           Paths.get("etc", "custom.properties"),
           Paths.get("etc", "config.properties"));
 
   private static final List<Path> OPTIONAL_SYSTEM_PATHS =
-      ImmutableList.of( //
-          Paths.get("etc", "users.properties"),
-          Paths.get("etc", "users.attributes"),
-          Paths.get("etc", "pdp", "ddf-metacard-attribute-ruleset.cfg"),
-          Paths.get("etc", "pdp", "ddf-user-attribute-ruleset.cfg"),
-          Paths.get("etc", "org.codice.ddf.admin.applicationlist.properties"),
-          Paths.get("etc", "fipsToIso.properties"),
-          Paths.get("etc", "log4j2.xml"),
-          Paths.get("etc", "certs", "meta"),
-          Paths.get("etc", "certs", "1"),
-          Paths.get("bin", "karaf"),
-          Paths.get("bin", "karaf.bat"));
+      ImmutableList.<Path>builder()
+          .addAll(UPGRADEABLE_SYSTEM_PATHS)
```

Comment:
❓Any reason we extracted these two paths to a constant but not the others?

Reply:
Yes. Nice find.

Reply:
Because only those 2 files are considered upgradeable, but they should also be included in this list of files that are exported/imported when on the same version.

Reply:
I decided to remove the use of the `UPGRADEABLE_SYSTEM_PATHS` constant here and just use the raw file paths instead, as it was before.